### PR TITLE
Multiple sale units

### PR DESCRIPTION
**Pull request by cloveras**
https://vippsas.slack.com/archives/C01BQJTPD09/p1613135939072600

### CODE REVIEW

File: vipps-ecom-api-faq.md [hunk 1]
```diff
@@ -853,9 +854,44 @@ to the requirements.
...
+It's possible to use the Vipps eCom API for several different types of payments.
+
+Let's say you run a bookstore. You can then Vipps eCom API in different ways:
+
+1. A webshop that sells physical books:
+   Vipps eCom API with "reserve capture", since you can not capture the payment before the book is shipped.
+2. A webshop that sells digital, downloadable books that are immediately available:
+   Vipps eCom API with "direct capture".
+3. A physical store where customers buy physical books in person:
+   Vipps eCom API with "reserve capture", possibly integrated with the POS.
```
Comment (tht13) [inline vipps-ecom-api-faq.md:868]: should this be direct?

File: vipps-ecom-api-faq.md [hunk 2]
```diff
@@ -853,9 +854,44 @@ to the requirements.
...
+4. A physical store where customers can buy physical books by scanning a
+   QR code in the window, and have the physical book delivered by mail:
+   Vipps eCom API with "reserve capture", since you can not capture the payment before the book is shipped.
+
+The regulatory requirements are different for different types of purchases.
+One major difference is if the cardholder is physically present and
+"can look the seller in the eye" while making the payment.
+
+Vipps need to do more thorough "Know Your Customer" (KYC) and compliance checks
```
Comment (tht13) [inline vipps-ecom-api-faq.md:877]: Grammar

"Vipps needs to do"

File: vipps-ecom-api-faq.md [hunk 3]
```diff
@@ -853,9 +854,44 @@ to the requirements.
...
+for some of the examples above. This must be done per sale unit.
+Vipps is also required to have the correct MCC
+([Merchant Category Code](https://en.wikipedia.org/wiki/Merchant_category_code))
+for each sale unit.
+
+Because of this, merchants must use separate sale units for separate types
+of purchases. This also has some benefits:
+
+- Each sale unit can have its own name presented to the user in Vipps
```
Comment (tht13) [inline vipps-ecom-api-faq.md:886]: Each sale unit has its own name

File: vipps-ecom-api-faq.md [hunk 4]
```diff
@@ -853,9 +854,44 @@ to the requirements.
...
+- Each sale unit can have its own settlement account
```
Comment (tht13) [inline vipps-ecom-api-faq.md:887]: Each sale unit can have its own settlement account, or sharing a single account across multiple sale units is available at request

File: vipps-ecom-api-faq.md [hunk 5]
```diff
@@ -853,9 +854,44 @@ to the requirements.
...
+- Each sale unit gets separate transaction logs
```
Comment (tht13) [inline vipps-ecom-api-faq.md:888]: Each sale unit has separate

File: vipps-ecom-api-faq.md (hunk 1 above)
Comment (cloveras) [reply vipps-ecom-api-faq.md]: Do you mean item 3? Yes, it shoyld probably be. Changing.

**Pull request merged**